databricks/koalas/series.py

```diff
@@ -668,6 +668,80 @@ def ne(self, other):
         """
         return (self != other).rename(self.name)
 
+    def between(self, left, right, inclusive=True):
+        """
+        Return boolean Series equivalent to left <= series <= right.
+        This function returns a boolean vector containing `True` wherever the
+        corresponding Series element is between the boundary values `left` and
+        `right`. NA values are treated as `False`.
+
+        Parameters
+        ----------
+        left : scalar or list-like
+            Left boundary.
+        right : scalar or list-like
+            Right boundary.
```

Comment:
Could you add a test if we can support list-like?

Reply:
Btw, pandas only support `scalar` value? https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.between.html

Reply:
https://github.com/pandas-dev/pandas/issues/28435

Reply:
cool!
Then shall we add tests for them?

Reply:
```
>>> a = ks.Series([1, 2, 3])
>>> b = ks.Series([4, 5, 6])
>>> a > b

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/harutaka/Desktop/projects/koalas/databricks/koalas/base.py", line 66, in wrapper
    return align_diff_series(apply_func, self, *args, how="full")
  File "/Users/harutaka/Desktop/projects/koalas/databricks/koalas/utils.py", line 255, in align_diff_series
    combined = combine_frames(this_series.to_frame(), *cols, how=how)
  File "/Users/harutaka/Desktop/projects/koalas/databricks/koalas/utils.py", line 116, in combine_frames
    raise ValueError("Cannot combine column argument because "
ValueError: Cannot combine column argument because it comes from a different dataframe
```

I got this error when I tried to compare two different series.

Reply:
This works.

```python
>>> df = ks.DataFrame({
    'a': [1, 2, 3],
    'b': [4, 5, 6],
    'c': [7, 8, 9]
})

>>> df.a < df.b
0    True
1    True
2    True
Name: a, dtype: bool

>>> df.b < df.c
0    True
1    True
2    True
Name: b, dtype: bool

```

Reply:
For different dataframe, you can add the tests under `test_ops_on_diff_frames.py`. `compute.ops_on_diff_frames` option allows operations on different DataFrames.